src/CodeFormatter/FormatOptions.cs

```diff
@@ -11,11 +11,17 @@ namespace CodeFormatter
     [Verb("format", HelpText = "Apply code formatting rules and analyzers to specified project.")]
     public class FormatOptions
     {
-        [Option(
-            't', "target", 
+        [Value(
```

Comment:
This change drops the requirement to prefix a SLN or project file target with -t or --target. All targets have to precede any other arguments on the command-line now, however.